lib/ASTGen/Sources/ASTGen/DeclAttrs.swift

```diff
@@ -436,24 +436,25 @@ extension ASTGenVisitor {
   ///   @_alignment(8)
   ///   ```
   func generateAlignmentAttr(attribute node: AttributeSyntax) -> BridgedAlignmentAttr? {
-    guard
-      let arg = node.arguments?.as(TokenSyntax.self)
-    else {
-      print("Not a token")
-      // TODO: Diagnose.
```

Comment:
Should this comment be preserved? Here and all the case below

Reply:
`generateWithLabeledExprListArguments` and `generateConsuming*` will diagnose if the input is unexpected.